lib/includes/json-helper.js

```diff
@@ -217,8 +218,12 @@ export const FHIRHelper = {
 
   /**
    * Given an SHR object, return a Reference that points to that object, and add the object to the given list.
+   *
+   * @param {Object} shrObject - the SHR object to reference
+   * @param {List<Object>} referenceList - the list of objects that are referenced
+   * @returns {string} the reference
```

Comment:
Should be `{Reference}` not string. That's really my bad for not commenting the function in the first place. Same thing in `createReferenceWithoutObject` below

Reply:
Well, now that I look at it, it's pretty obvious from the code -- so really, it's my bad.  Either copy/paste error or I just made a bad assumption without verifying it.  I'll fix it.